pyaci/core.py

```diff
@@ -128,7 +128,9 @@ def _performRequest(self, method, format=None, needData=False, **kwargs):
 
         req = Request(method, url, data=data)
         prepped = rootApi._session.prepare_request(req)
-        self._x509Prep(rootApi, prepped, data)
+        # never use certificate for subscription requests
+        if "subscription" not in kwargs.keys():
```

Comment:
Idiomatic way of writing this is in Python in `if "subscription" not in kwargs:`.

``` python
>>> x = {'a': 1, 'b': 2}
>>> 'a' in x.keys()
True
>>> 'a' in x
True

Reply:
Can you please fix this, and rebase? Even though it is minor I want the rebase is because that way you can exercise the new circleci integration for this PR. Thanks!

Reply:
done - circleci tests are still failing, is this specific to provided commit or some other issue?